tools/indexer/example/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
Please, write some README with the minimal setup where you can test things out on localnet and testnet using `near-shell`